synth.py

```diff
@@ -0,0 +1,32 @@
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+from pathlib import Path
+import subprocess
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+
+versions = ['v1']
+
+for version in versions:
+    library = gapic.node_library(
+        'container', version,
+        config_path="/google/container/"
+                    "artman_container.yaml")
```

Comment:
same thing here as another pr. the yaml isn't versioned so if you haded v2, you would regenerate the same yaml as this is written.